save-demo/src/main/kotlin/com/saveourtool/save/demo/service/AbstractDemoService.kt

```diff
@@ -2,17 +2,22 @@ package com.saveourtool.save.demo.service
 
 import com.saveourtool.save.demo.DemoAdditionalParams
 import com.saveourtool.save.demo.DemoResult
+import com.saveourtool.save.demo.runners.cli.CliRunner
+import org.springframework.stereotype.Service
 
 /**
  * Abstract service interface for different demonstration services
  */
-interface AbstractDemoService<in P : DemoAdditionalParams, out R : DemoResult> {
+@Service
```

Comment:
I don't think abstract class should be annotated as a Spring component

Reply:
Makes sense

Reply:
Removed annotation from here